recipe/meta.yaml

```diff
@@ -1,46 +1,49 @@
 {% set name = "graphql-relay" %}
-{% set version = "0.4.5" %}
-{% set bundle = "tar.gz" %}
-{% set hash_type = "sha256" %}
-{% set hash = "2716b7245d97091af21abf096fabafac576905096d21ba7118fba722596f65db" %}
-{% set build = 1 %}
+{% set version = "2.0.0" %}
 
 package:
-  name: {{ name|lower }}
+  name: {{ name }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.{{ bundle }}
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ bundle }}
-  {{ hash_type }}: {{ hash }}
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: 7fa74661246e826ef939ee92e768f698df167a7617361ab399901eaebf80dce6
+    folder: dist
+  - url: https://github.com/graphql-python/{{ name }}-py/archive/v{{ version }}.tar.gz
```

Comment:
why do you need to manually pull this in?

Reply:
So the one thing that this might do is confuse the bot :(  You may want to ONLY use the GH tarball in this case

Reply:
@bollwyvl are you able to refactor as per @mariusvniekerk's recommendation? If you give me access to your fork, I can also try to do it.

Reply:
cheers, sorry i missed that feedback. updated to just use the tagged github tarball.